internal/proxy/plan_parser.go

```diff
@@ -858,7 +858,7 @@ func (pc *parserContext) handleExpr(nodeRaw *ant_ast.Node) (*planpb.Expr, error)
 	case *ant_ast.BinaryNode:
 		return pc.handleBinaryExpr(node)
 	default:
-		return nil, fmt.Errorf("unsupported node (%s)", node.Type().String())
+		return nil, fmt.Errorf("unsupported node")
```

Comment:
should print the original expression?

Reply:
original expr has been printed out in upper level